pages/pipelines/notifications.md.erb

```diff
@@ -97,39 +97,63 @@ Basecamp notifications happen at the following [events](/docs/apis/webhooks#even
 * `build finished`
 * `build skipped`
 
-## Slack Channel and Direct Messages
+## Slack channel and direct messages
 
-If you need fine-grained control over your notifications, or want to send notifications triggered by non-build events, the [Slack Notification Service](/docs/integrations/slack) can be extended using the `notify` attribute.
+You can set notifications:
 
-If you only need to send notifications on build status events, you can set this up in Buildkite using your [Slack Notification Service's](/docs/integrations/slack) 'Build State Filtering' settings.
+* On build status events in the Buildkite UI by using your Slack Notification Service's 'Build State Filtering' settings.
+* On step status and other non-build events by extending the Slack Notification Service using the `notify` attribute to your `pipeline.yml`.
 
-Before adding a `notify` attribute to your `pipeline.yml`, ensure an organization admin has set up a [Slack integration](/docs/integrations/slack) for the channel or user that you want to post to. For detailed information about setting up a Notification Service, see the [Slack integration page](/docs/integrations/slack).
+Before adding a `notify` attribute to your `pipeline.yml`, ensure an organization admin has set up a [Slack integration](/docs/integrations/slack) for the channel or user that you want to post to. Buildkite Enterprise users can also check the ['Manage Notifications Services'](https://buildkite.com/organizations/~/member-permissions) checkbox to create, edit, or delete notification services. For detailed information about setting up a Notification Service, see the [Slack integration page](/docs/integrations/slack).
```

Comment:
I would also add that if they rename the channel or make a modification (like changing it from public to private), they need to set up a new integration.

Reply:
Thank you so much for your review, @pzeballos!